.githooks/pre-commit

```diff
@@ -1,15 +1,15 @@
 #!/bin/sh
 
-echo "Running detekt check"
-OUTPUT="/tmp/detekt-$(date +%s)"
-./gradlew detekt > $OUTPUT
+echo "Running ktlint check"
+OUTPUT="/tmp/ktlint-$(date +%s)"
+./gradlew lintKotlin > $OUTPUT
```

Comment:
```suggestion
./gradlew formatKotlin > $OUTPUT
```
So we would have this auto-formatted and notified only when it's failed. It's not a change request

Reply:
@pawelpasterz this will change. Looking into alternatives.

Reply:
Changed.